style.css

```diff
@@ -0,0 +1,128 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+body{
+    min-height: 100vh;
+    background-color: #d3d3d3;
+    font-family: 'Roboto', sans-serif;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+.container{
+    position: absolute;
+    left: 50%;
+    transform: translateX(-50%);
+    perspective: 2000px;
+    perspective-origin: 50% 50%;
```

Comment:
What does this property do ? @what does this property do @murtazshaikh

Reply:
It determines the position from where the viewer is looking, in this case 50% 50% means we are looking from center.